### ISSUE TITLE
KoreBuild with .NET Core 1.0.1

### ISSUE THREAD

**Issue by henkmollema** [aspnet/KoreBuild#113]
Is it possible to update KoreBuild to use `1.0.0-preview2-003131` (1.0.1)? Could we bump the `cli.version.*` guys?

**Comment by f7q** [comment]
Hi, I tried that.

1.`.NETCore 1.0.1 SDK` Installed.
2.Fork aspnet/KoreBuild
3.master branch on `cli.version.*`, I changed this files.

``` diff
- 1.0.0-preview2-003121
+ 1.0.0-preview2-003131
```

4.My local project Script file change.
- `build.ps1`
- `build.sh`

``` diff
- https://github.com/aspnet/KoreBuild/archive/1.0.0.zip
+ https://github.com/f7q/KoreBuild/archive/1.0.1-bata2.zip
```

Checked state of the CI runner result.
- [x] travis is OK
- [x] AppVeyor is OK
- [x] CircleCI is OK

@pranavkm
Could you put a new tag `KoreBuild-1.0.1`?

**Comment by pranavkm** [comment]
@davidfowl wanted us to stick to the RTM released version of the CLI. @davidfowl, do you have any concerns about updating the CLI?

**Comment by henkmollema** [comment]
This is fixed with 45b1eec7064568d1a70bf6e9bde3955cf5f24698.
